kotsadm/pkg/handlers/troubleshoot.go

```diff
@@ -449,6 +407,7 @@ func UploadSupportBundle(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
@laverya can you help me with this one? i cant find it called from the ui and it looks like you wrote it

Reply:
I'm not sure either right now - let me take a look